src/bioregistry/curation/literature.py

```diff
@@ -0,0 +1,44 @@
+"""Utilities for working with the data produced by the semi-automated curation workflow."""
+
+import enum
+
+import click
+
+__all__ = [
+    "CurationRelevance",
+    "COLUMNS",
+]
+
+COLUMNS = [
+    "pmid",
+    "relevant",
+    "relevancy_type",
+    "orcid",
+    "date_curated",
+    "notes",
+    "pr_added",  # links back to the PR where curations were done
+]
+
+
+class CurationRelevance(str, enum.Enum):
+    """An enumeration for curation relevance."""
+
+    #: A resource for new primary identifiers
+    new_prefix = enum.auto()
+    new_provider = enum.auto()
```

Comment:
Add docstrings for all of these. You can see the results with

```shell
tox -e docs
open docs/build/html/curation.html
```

Reply:
Doesn't this mean we now have to have two separate descriptions of the same concept that have to be in synç? As I commented before, the vocabulary at this stage was meant to be preliminary and will evolve while we explore the initial batch of papers. Once we are confident in a good set that covers all relevant types we come across, we can formalize it as a closed vocabulary and start testing for it.

Reply:
@nagutm let's add the docstrings here to complete this part